init.sh

```diff
@@ -214,7 +214,7 @@ export MAS_JDBC_CERT_LOCAL_FILE=$GIT_REPO_HOME/db.crt
 export MAS_CLOUD_AUTOMATION_VERSION=1.0
 export MAS_DEVOPS_COLLECTION_VERSION=11.0.0
 export MAS_APP_CHANNEL=8.4.x
-if [ -z "$EXISTING_NETWORK" ]; then
+if [[ (-z "$EXISTING_NETWORK") && (-z "$EXS_OCP_URL") ]]; then
```

Comment:
Hi @santoshjpawar, this condition we had added for bastion host creation control. I think it is not needed since you've already added the following check in deploy.sh
`&& ($OPENSHIFT_USER_PROVIDE == "false")`

```
if [[ ($new_or_existing_vpc_subnet == "new") && ($OPENSHIFT_USER_PROVIDE == "false") ]]; then
    ./create-bastion-host.sh
    retcode=$?
    if [[ $retcode -ne 0 ]]; then
      log "Bastion host creation failed in Terraform step"
      exit 22
    fi
  fi
```
please suggest

Reply:
Correct, this conditions is not needed

Reply:
Done.